arm-apimanagement/2016-07-07/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1236 @@
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "ResourcesMoveInfo": {
+      "properties": {
+        "resources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The ids of the resources."
+        },
+        "targetResourceGroup": {
+          "type": "string",
+          "description": "The target resource group."
+        }
+      },
+      "description": "Parameters of move resources."
+    },
+    "CertificateInformation": {
+      "properties": {
+        "expiry": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Expiration date of the certificate. The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.\n"
+        },
+        "thumbprint": {
+          "type": "string",
+          "description": "Thumbprint of the certificate."
+        },
+        "subject": {
+          "type": "string",
+          "description": "Subject of the certificate."
+        }
+      },
+      "required": [
+        "expiry",
+        "thumbprint",
+        "subject"
+      ],
+      "description": "SSL certificate information."
+    },
+    "HostnameConfiguration": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Hostname type.",
+          "enum": [
+            "Proxy",
+            "Portal",
+            "Management",
+            "Scm"
+          ],
+          "x-ms-enum": {
+            "name": "HostnameType",
+            "modelAsString": false
+          }
+        },
+        "hostname": {
+          "type": "string",
+          "description": "Hostname."
+        },
+        "certificate": {
+          "$ref": "#/definitions/CertificateInformation",
+          "description": "Certificate information."
+        }
+      },
+      "required": [
+        "type",
+        "hostname",
+        "certificate"
+      ],
+      "description": "Custom hostname configuration."
+    },    
+    "VirtualNetworkConfiguration": {
+      "properties": {
+        "vnetid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Virtual network id."
+        },
+        "subnetname": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Subnet name."
+        },
+        "subnetResourceId": {
+          "type": "string",
+          "description": "Subnet Resource Id."
+        },
+        "location": {
+          "type": "string",
+          "description": "Virtual network location name."
+        }
+      },
+      "description": "Configuration of a virtual network."
+    },
+    "AdditionalRegion": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location name."
+        },
+        "skuType": {
+          "type": "string",
+          "description": "Sku type in the location.",
```

Comment:
Made Skutype a required parameter in the additional location. We have kept the flexibility to have  different skuType in additional location than the one specified in ApiServiceManageDeploymentParameters.

---
In reply to: [83512579](https://github.com/Azure/azure-rest-api-specs/pull/573#discussion_r83512579) [](ancestors = 83512579)